File: modul4/Python/PRAK402-2310817210023-RifkyPutraMahardika.py
angka = int(input("Input : "))
# Untuk angka ganjil
print("Output : ")
for i in range(1, angka+1, 2):
    print(i, end=" ")

# Untuk angka genap
print()
for i in range(angka, 1, -2):
    print(i, end=" ")

File: modul4/C/PRAK402-2310817210023-RifkyPutraMahardika.c
#include <stdio.h>
int main() {
    int angka;
    printf("Input : ");
    scanf("%d", &angka);

    //Untuk angka ganjil
    printf("Output : \n"); 
    for(int i=1; i<=angka; i++){
        if(i % 2 != 0){
            printf("%d ", i);
        }
    }

    //Untuk angka genap
    printf("\n");
    for(int i=angka; i>=2; i--){
        if(i % 2 == 0){
            printf("%d ", i);
        }
    }
}